timelock-impl/src/main/metrics/metadata.yml

```diff
@@ -0,0 +1,20 @@
+options:
+  javaPackage: 'com.palantir.atlasdb.timelock.metrics'
```

Comment:
We should also think about how we can track # of bytes as well.

Reply:
Number of bytes is a bit trickier since we potentially have to iterate over all metadata objects. For SKAPC, we know an upper bound for the number of bytes per `ChangeMetadata` object (maximum key size). Do we want the exact number anyways?

Reply:
Yeah I was thinking of just tracking it when we do an `add`. It would be a meter metric, so it's more of the _rate_ than total size.

Reply:
We could also do a counter, where we minus once we remove something from the sliding window